src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/products/product.yml

```
@@ -520,3 +520,11 @@ admin_products_quantity:
     shopId: \d+
   options:
     expose: true
+
+admin_category_tree:
+  path: /category-tree
+  methods: [ GET ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\Product\ProductController::categoryTreeAction
+    _legacy_controller: AdminProducts
+    _legacy_link: AdminProducts
```

```suggestion
    _legacy_link: AdminProducts:getCategoryTree
```
It's important to indicate the related action in the legacy link, it prevents this action from being confused with the default index action (when no action is defined it is assumed it is the index)